src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -39,13 +96,33 @@ public PlayerEditor() {
 
         add(new JScrollPane(playerInfoList,
                 JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER), "span, grow");
+
+        playerInfoList.addListSelectionListener(event -> SwingUtilities.invokeLater(() -> {
+            if (!event.getValueIsAdjusting()) {
```

Comment:
```suggestion
            if (event.getValueIsAdjusting()) return;
```
You can simplify these all-encompasing ifs with just returning on the oposite case. That way you avoid indenting-in the whole method